code/src/pages/MovieDetail.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { NotFound } from '../components/NotFound';
+import { Loader } from '../components/Loader';
+import { BackButton } from '../components/BackButton';
+
+import '../styles/MovieDetail.css';
+
+export const MovieDetail = () => {
+  const [movie, setMovie] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const { movieId } = useParams();
+
+  useEffect(() => {
+    fetch(
+      `https://api.themoviedb.org/3/movie/${movieId}?api_key=a4952259f6d389d2957bfec34fa69938&language=en-US`
```

Comment:
I think it's really good that you want to look into making the key not visible 👍 
We should not give away sensitive content

Reply:
And now it's done!